requirements/docs.txt

```diff
@@ -4,6 +4,7 @@
 -r dev.txt
 -r extra.txt
 
+pyyaml
```

Comment:
should use ruamel.yaml istead it's much better maintained. otoh, would it not be worth using *shudder* json so that we don't need this dep?

Reply:
If you or @dpshelio want to work on that. I will stick with the lazy approach.